### ISSUE TITLE
Metric Conversion

### ISSUE THREAD

**Issue by reuvygroovy** [angela-d/brain-dump#1]
Hi,

On your explanation you write:
The Multiplier option in the PRTG plugin had to be set to **0.1** to display a correct reading, but if you look carefully at the screenshot underneath you wrote in the multiplier field **.01**. Not sure which value is correct.

Also, I tried both (0.1, 0.01) and neither matched the values I saw in PRTG. The only way I was able to align the metrics was by seeing that I was recieving bytes, and then I had to divide by 1048576 in order to receive a value in MB. For example the raw data I received was "671973628" and when viewing the value in PRTG I saw the value "641 MByte".

**Comment by angela-d** [comment]
Good catch, that's a typo!

I am using `.01`  (updated the typo in my notes, too)

I think PRTG's default is kilobit, but if the channel you're obtaining the data from has a custom calculation built in, it might be processed differently. (Just guessing, I've not encountered such a thing myself, yet)

If you go into the channel settings, do you see any additional options? (For instance, is a value lookup specified?)

> The only way I was able to align the metrics was by seeing that I was recieving bytes, and then I had to divide by 1048576 in order to receive a value in MB. 

Where did you find to do that?  I was looking for a spot to enter something like this, but couldn't make anything work.

This particular area is very weird.  There's no explanation why its handled the way it is in the plugin.  I came across an issue on neuralfraud's Github, randomly tried it and compared the graph to PRTG and got matches.

I still don't fully understand why it works the way it does. :)

***
Edit: Also check what version you're on (of the plugin), I just got 4.0.4 working (after changing the channel name targets of graphs) and I notice a significant difference in measurement values, so .01 is no longer a proper calculation.

Update again:
To get a more accurate measurement, my graphs calculation is now at `.139` instead of `.01` and I changed the Visualization unit to **bytes/sec** - it's off by approximately 1-2MB from PRTG, but I can live with that.
When I have more time to mess with it, I will get a more accurate computation.  For now, `.139` is giving "good enough" results.

My particular Grafana display is for an IT overview.. I wouldn't necessarily recommend doing a guesstimate rounding like this for something mission critical that could put lives at risk, or anything.

**Comment by reuvygroovy** [comment]
Hi
When browsing the data returned from within PRTG to Grafana, I saw the raw data. by trial and error, I figured it out that it was in Bytes, and in the tables below I wanted Mb/sec, so I ran a conversion of multiplying the bytes received by a multiplier of 0.000008 and that gave me the exact value. I don't personally manage PRTG in house, but am rather in charge of Grafana. Could be this was something unique and special in my environment. Thanks for the help!
